test/TestBulkRenewal.js

```diff
@@ -0,0 +1,108 @@
+const ENS = artifacts.require('@ensdomains/ens/ENSRegistry');
+const HashRegistrar = artifacts.require('@ensdomains/ens/HashRegistrar');
+const PublicResolver = artifacts.require('@ensdomains/resolver/PublicResolver');
+const BaseRegistrar = artifacts.require('./BaseRegistrarImplementation');
+const ETHRegistrarController = artifacts.require('./ETHRegistrarController');
+const DummyOracle = artifacts.require('./DummyOracle');
+const StablePriceOracle = artifacts.require('./StablePriceOracle');
+const BulkRenewal = artifacts.require('./BulkRenewal');
+var Promise = require('bluebird');
+
+const namehash = require('eth-ens-namehash');
+const sha3 = require('web3-utils').sha3;
+const toBN = require('web3-utils').toBN;
+
+const NULL_ADDRESS = "0x0000000000000000000000000000000000000000"
+const ETH_LABEL = sha3('eth');
+const ETH_NAMEHASH = namehash.hash('eth');
+
+const advanceTime = Promise.promisify(function(delay, done) {
```

Comment:
can we either use the library for this already or make a PR to do so later. `test-utils` is already used in all other tests.

Reply:
Then it will be fixed in a future PR. As a policy, requiring every issue to be fixed in any file that is touched by it just serves to discourage touching that code at all, and confuses the purpose and content of a PR.